plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -545,15 +545,24 @@ class NetworkingFormSection extends mixin(StoreMixin) {
   }
 
   getVirtualNetworks() {
+    const mesosContainer =
+      findNestedPropertyInObject(this.props.data, `container.type`) === "MESOS";
+
     return VirtualNetworksStore.getOverlays()
       .mapItems(overlay => {
         const name = overlay.getName();
 
         return {
+          enabled: overlay.info.enabled,
+          subnet6: overlay.getSubnet6(),
           text: name,
           value: `${CONTAINER}.${name}`
         };
       })
+      .filterItems(
+        virtualNetwork =>
+          !mesosContainer || !virtualNetwork.subnet6 || !virtualNetwork.enabled
```

Comment:
`filter` should return `true` to keep elements, shouldn't it? 

I might not being clear enough, sorry.

We want to remove `enabled: false` items regardless of the network type and container type. Plus we want to remove `subnet6` networks for `mesosContainer`.

We also want to fix pods `MultiContainerNetworkingFormSection#getVirtualNetworks` pods _always_ use UCR so we should always remove `subnet6` networks and of course remove `enabled: false` as well.